scripts/local.sh

```diff
@@ -10,11 +10,17 @@ TOP_DIR=$(cd $(dirname "$0") && pwd)
 
 FULL_PROJECT_CONFIG_PATH=${FULL_PROJECT_CONFIG_PATH:-/home/rocket-man/project-config-third-party}
 
-if [ -d "$FULL_PROJECT_CONFIG_PATH" ]; then
+if [ ! -d "$FULL_PROJECT_CONFIG_PATH" ]; then
     echo "Could not run through local.sh because FULL_PROJECT_CONFIG_PATH is not set to a valid path"
     exit 0
 fi
 
+# Try to install requirements
+sudo pip install diskimage-builder
+sudo pip install ansible
```

Comment:
I think it would probably be a good idea to specify a specific version of ansible, given we’ve run into problems with compatibility in the past